src/backend/x11/buffer.rs

```diff
@@ -52,6 +52,7 @@ impl From<ConnectionError> for CreatePixmapError {
 
 #[derive(Debug)]
 pub struct Pixmap {
+    // TODO: Consider future x11rb WindowWrapper
```

Comment:
The release came faster than I expected: https://docs.rs/x11rb/0.9.0/x11rb/protocol/xproto/struct.WindowWrapper.html

Reply:
The fact I somewhat rely on shared references means I would need to look over the implementation to use the wrappers since those have lifetimes.

I'll consider it once I have polished everything up on api side and some parts of implementation.

Reply:
Pixmap was moved to wrapper, window is the contentious part.